mon-pix/app/components/challenge-statement.js

```diff
@@ -1,63 +1,55 @@
-import { classNames } from '@ember-decorators/component';
-import { action, computed } from '@ember/object';
+import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 import config from 'mon-pix/config/environment';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
-import _ from 'lodash';
+import Component from '@glimmer/component';
 
-@classic
-@classNames('challenge-statement')
 export default class ChallengeStatement extends Component {
   @service mailGenerator;
 
-  challenge = null;
-  assessment = null;
+  @tracked selectedAttachmentUrl;
 
-  @computed('challenge.attachments')
-  get selectedAttachmentUrl() {
-    return  _.get(this.challenge, 'attachments.firstObject');
+  constructor() {
+    super(...arguments);
+    if (this.args.challenge.hasMultipleAttachments) {
+      this.selectedAttachmentUrl = this.args.challenge.attachments.firstObject;
+    }
```

Comment:
Sortir dans une fonction `_initRadioButtons` + voir si on peut se passer du if grâce à 
`_.get(this.challenge, 'attachments.firstObject')`